similarity_api/src/application.py

```diff
@@ -1,10 +1,37 @@
+import logging
+import atexit
+
 from flask import Flask
 from db.db import db
+from sim.compute import cache_scores as cache
+from flask_apscheduler import APScheduler
+
+scheduler = APScheduler()
 
 def create_app():
     app = Flask(__name__, static_folder='static')
+
     app.config.from_pyfile("settings.py")
+
     db.init_app(app)
+
     from routes.create import create_routes
     create_routes(app)
+
+    scheduler.init_app(app)
+    scheduler.start()
+    # scheduler.add_job(func=cache_scores, id='cache', trigger='interval', seconds=10, timezone='EST')
+    scheduler.add_job(func=cache_scores, id='cache', trigger='cron', minute='0', hour='3', timezone='GMT', replace_existing=True, max_instances=1)
+
+    atexit.register(lambda: scheduler.shutdown(wait=False))
+
+    # By default, APScheduler logs are a bit noisy.
+    logging.getLogger("apscheduler.scheduler").setLevel(logging.DEBUG)
```

Comment:
Can logging level be set by an ENV and default to this value

Reply:
287ad2e2165a6f80c3ef6c424dfa90b3fc3e5609